src/app/components/confirmBtcTransactionComponent/index.tsx

```diff
@@ -343,9 +350,9 @@ function ConfirmBtcTransactionComponent({
             recipients?.map((recipient, index) => (
               <RecipientComponent
                 recipientIndex={index + 1}
-                address={recipient?.address}
-                value={satsToBtc(recipient?.amountSats).toString()}
-                totalRecipient={recipients?.length}
```

Comment:
why do we remove the undefined checks here?

Reply:
the undefined check was unnecessary here. The undefined check should be with the recipient array which is added. 
Here we iterate over the recipient array and every object in the array will have an address and amountSats since these parameters are not optional.